test/commands/extended/get_bundles_test.py

```diff
@@ -353,7 +356,7 @@ def test_multiple_transactions(self):
           b'DRXICGYDGSVPXFTILFFGAPICYHGGJ9OHXINFX9999'
         ),
     )
-
+    print(response['bundles'][0].as_tryte_strings())
```

Comment:
Oops (:

Reply:
Yea I had to do some investigating on that one...